docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```diff
@@ -1,90 +1,189 @@
+
+This will take you to the Teleport Discover UI:
+
+![Select resource
+type](../../../img/cloud/getting-started/select-resource-type.png)
+
+Leave **Server** selected and click **Next**.
+
+You will see a script that you can copy from the Teleport Web UI:
+
+![Configure resource](../../../img/cloud/getting-started/configure-resource.png)
+
+Copy the script and paste it into the shell you opened in the previous step.
+Remove `sudo` from the start of the script so you can run it on your container.
+It should look something like this:
+
+```text
+$ bash -c "$(curl -fsSL https://TENANT_SUBDOMAIN.teleport.sh/scripts/SCRIPT_NAME/install-node.sh)" 
+```
+
+Leave the script running in your shell. Teleport will run in the foreground. You
+will see a message in the Web UI that your new server connected successfully:
+
+![Connected
+successfully](../../../img/cloud/getting-started/successfully-connected.png)
 
-Install client libraries:
 
-<Tabs>
-  <TabItem label="MacOS">
-(!docs/pages/includes/enterprise/install-macos-ent.mdx!)
-  </TabItem>
-  <TabItem label="Windows">
-(!docs/pages/includes/install-windows.mdx!)
-  </TabItem>
+Choose the OS user you will assume when you authenticate to the server, in this
+case, `root`:
 
-  <TabItem label="Linux">
-    ```code
-    $ curl -O https://cdn.teleport.dev/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    # verify signature 
-    $ echo "$(curl  https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz.sha256)" | sha256sum --check
-    $ tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    $ cd teleport-ent
-    $ sudo ./install
-    ```
-  </TabItem>
-</Tabs>
+![Choose your user](../../../img/cloud/getting-started/set-up-access.png)
 
-Login into Teleport and test the connection:
+From there, you can test the connection and access the server. Click **Start
+Session** to connect:
+
+![Start session](../../../img/cloud/getting-started/test-connection.png)
+
+<Notice type="tip">
+
+If you get an error connecting to your container, interrupt the `teleport`
+process in your container's shell with **Ctrl-C**, then run `teleport start` to
+restart the process.
+
+</Notice>
+
+You should see a terminal prompt in your browser. Your Teleport Enterprise Cloud
+account proxies your SSH connection through the Teleport Proxy Service, which
+connects to your container via a reverse tunnel. 
+
+## Step 3/4. Access your server from the command line
+
+Log in to your Teleport Enterprise Cloud tenant via the command line, filling in
+the URL of your tenant and the name of your Teleport user:
+
+```code
+$ tsh login --proxy=<Var name="tenant-url" /> --user=<Var name="username" />
+```
+
+List the servers your Teleport user can access. You should see the name of the
+container you just registered:
 
 ```code
-# tsh logs you in and receives short-lived certificates
-$ tsh login --proxy=myinstance.teleport.sh --user=email@example.com
 $ tsh ls
```

Comment:
Why remove the example output? Should we include something like:

```
Node Name    Address    Labels
------------ ---------- ----------------------------------------------------------------------------------------
(=some.var=) ⟵ Tunnel   hostname=(=some.var=),teleport.internal/resource-id=(=some.other.var=)
```

So that first time users know what to expect?